airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceStandardTest.java

```diff
@@ -87,6 +88,8 @@
   private static final Set<String> TEST_SCHEMAS = ImmutableSet.of(SCHEMA_NAME, SCHEMA_NAME2);
 
   private static final String TABLE_NAME = "id_and_name";
+  private static final String TABLE_NAME_WITHOUT_PK = "id_and_name_without_pk";
+  private static final String TABLE_NAME_FULL_NAMES = "full_names";
```

Comment:
can we call this multi column pk or something? i had trouble parsing this name. wasn't until i read the create table query that i understood it.